ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveMetaStoreClientWithLocalCache.java

```diff
@@ -61,7 +61,6 @@
 import org.apache.hadoop.hive.metastore.api.TableValidWriteIds;
 import org.apache.hadoop.hive.metastore.api.UniqueConstraintsRequest;
 import org.apache.hadoop.hive.metastore.api.UniqueConstraintsResponse;
-import org.apache.hadoop.hive.metastore.conf.MetastoreConf;
```

Comment:
Is this diff required?

Reply:
Since `HiveMetaStoreClientWithLocalCache.java` is in HS2, I thought I might as well clean this up and use `HiveConf` instead of `MetastoreConf`. Let me know if this doesn't make a lot of sense, and I can revert it.

Reply:
This file was moved from `metastore-common` to `ql` in https://github.com/apache/hive/commit/f15de94c617c4566c87293479463cd90437beed5#diff-011304448070f76deed9090288839d16ee84ded1fb575a324f11a31bc6bab1e7, 
so I think it makes sense to use `HiveConf` instead of `MetastoreConf`